README.md

```diff
@@ -76,6 +77,37 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
 
+## `chectl devfile:generate`
+
+generate and print a devfile to stdout given some Kubernetes resources and other Che workspaces features (project, language-support, commands etc...)
+
+```
+USAGE
+  $ chectl devfile:generate
+
+OPTIONS
+  -h, --help                 show CLI help
+  --command=command          Command to include in the workspace
+  --dockerimage=dockerimage  dockerimage component to include in the Devfile
+  --editor=editor            Specify the Che editor component. Currently supported editors: theia-next,theia-1.0.0
+  --git-repo=git-repo        Source code git repository to include in the workspace
+
+  --language=language        Add support for a particular language. Currently supported languages:
+                             java,typescript,go,python,c#
+
+  --name=name                Workspace name
+
+  --namespace=namespace      Kubernetes namespace where the resources are defined
+
+  --plugin=plugin            Che plugin to include in the workspace. The format is JSON. For example this is a valid Che
+                             Plugin specification: {"type": "TheEndpointName.ChePlugin", "alias": "java-ls", "id":
+                             "redhat/java/0.38.0"}
+
+  --selector=selector        label selector to filter the Kubernetes resources
```

Comment:
Maybe it makes sense to add example to make it clear that selector has format `key=value`:
`--selector=app.kubernetes.io/name=my-app`

Reply:
Done thanks for the review @sleshchenko